.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: "Unit Tests"
+
+on:
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  test:
+    name: Running Unit Tests
+    runs-on: ubuntu-latest
+    container: cfcli/cli-base:latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Run Tests
+        run: |
+          set -ex
+
+          go install github.com/onsi/ginkgo/ginkgo@latest
+
+          BINARY_VALIDATION=false bin/test
```

Comment:
What do you think if we would configure `BINARY_VALIDATION` on a step level?
https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsstepsenv